backend/MiniLcm/MultiString.cs

```diff
@@ -10,28 +10,36 @@ namespace MiniLcm;
 /// </summary>
 // convert to a dictionary of string to string
 [JsonConverter(typeof(MultiStringConverter))]
-public class MultiString
+public class MultiString: IDictionary
```

Comment:
What do we gain by making this an `IDictionary`?

Reply:
when JsonPatch applies changes it will work properly.